src/marqo/index.py

```diff
@@ -515,6 +515,10 @@ def verbosely_add_docs(i, docs):
         mq_logger.debug('completed batch ingestion.')
         return results
 
+    def get_settings(self) -> dict:
```

Comment:
Duplicate method. Looks like a merge conflict resolved incorrectly? Please keep the mainline version that passed `index_name`